static/js/zamboni/static_theme.js

```diff
@@ -202,17 +209,32 @@ $(document).ready(function() {
         }));
 
         function uploadDone(data) {
-            //console.log(data);
             if (!data.validation) {
                 setTimeout(function() {
                     $.ajax({
                         url: data.url,
                         dataType: 'json',
-                        success: uploadDone
+                        success: uploadDone,
+                        error: function (xhr, text, error) {
+                            // Fake the validation so we can display as an error.
+                            uploadDone({validation:{
+                                errors:1,
+                                messages:[{message:error}]
+                            }});
+                        }
                     });
                 }, 1000);
             } else {
-                $wizard.find('#submit-describe').submit();
+                if (data.validation.errors === 0 ) {
+                    $wizard.find('#submit-describe').submit();
+                } else {
+                    data.validation.messages.forEach(function(message) {
+                       $('.errorlist.validator').append($('<li>', {'html': message.message}));
+                       console.error(message);
```

Comment:
Stray `console.error()`

Reply:
I left it in on purpose - `message` exposes a bit more information than the text (`message.message`)... though not a lot more so it could be argued its superfluous.